src/main/java/seedu/club/model/ModelManager.java

```diff
@@ -311,13 +312,39 @@ public void addTaskToTaskList(Task toAdd) throws DuplicateTaskException {
         }
     }
 
+    @Override
+    public void assignTask(Task toAdd, Name name) throws MemberNotFoundException, DuplicateTaskException {
+        boolean found = false;
+        for (Member member : filteredMembers) {
```

Comment:
This will only search filtered list so if i use find to filter the list i might not find a member that was filtered out by the find command.

Reply:
Thanks for pointing it out! Will change it 👍